### DISCUSSION TITLE
Is opcache with jit a good idea with nextcloud?

### DISCUSSION THREAD

**Discussion by MrRinkana** [nextcloud/all-in-one#4266] [Questions] [answered]
Basically it seems like the php opcache JIT is having some problems:
https://github.com/php/php-src/issues/7817
https://github.com/php/php-src/pull/10206#issuecomment-1417597653
https://github.com/php/php-src/pull/10206#issuecomment-1375551589
Which is the reason its disabled by default.

In essence it seems like JIT should not be used in production, to my eyes.

The Nextcloud AIO uses opcache with JIT enabled.

My question is: are the above mentioned problems tested and not appearing in the case of running nextcloud? Or seen as a non-issue? I would like to know what the rationale is around having it enabled.

I like speed, so if it can safely be enabled in the case of nextcloud I would be happy, it just seems like it shouldn't currently.

**Comment by szaimen** [comment, marked as answer]
Hi, I have not seen any problem with jit yet when used with Nextcloud. So out of exprrience I'd say it is fine to be used with Nextcloud.
